addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1088,18 +1094,29 @@ var SnippetEditor = Widget.extend({
         // Remove the siblings that belong to a snippet in grid mode
         // and put the identified grid mode snippets in their own "selector".
         const selectorGrids = new Set();
-        if (rowEl.classList.contains('row')) {
-            if ($selectorSiblings) {
-                // Looping backwards because elements are removed, so the
-                // indexes are not lost.
-                for (let i = $selectorSiblings.length - 1; i >= 0; i--) {
-                    if (isPopup && !$selectorSiblings[i].closest('div.s_popup')) {
-                        // Removing the siblings that are outside the popup if
-                        // the grid item is in a popup.
-                        $selectorSiblings.splice(i, 1);
-                    } else {
-                        const gridSnippet = $selectorSiblings[i].closest('div.o_grid_mode');
-                        if (gridSnippet) {
+        if (rowEl.classList.contains('row') && $selectorSiblings) {
+            // Looping backwards because elements are removed, so the
+            // indexes are not lost.
+            for (let i = $selectorSiblings.length - 1; i >= 0; i--) {
+                if (isPopup && !$selectorSiblings[i].closest('div.s_popup')) {
+                    // Removing the siblings that are outside the popup if
+                    // the grid item is in a popup.
+                    $selectorSiblings.splice(i, 1);
+                } else {
+                    const gridSnippet = $selectorSiblings[i].closest('div.o_grid_mode');
+                    if (gridSnippet) {
+                        // The grid in which the dragged column is (if any).
+                        const targetGrid = rowEl.classList.contains('o_grid_mode') || rowEl.closest('div.o_grid_mode');
```

Comment:
Difficult to read: I don't know if this is a boolean or an element:
- If it is a boolean, find a better name and do `!!el.closest(...)`
- If it is an element, suffix the variable name by `El` (I guess it is an element here)